specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2024-09-01-preview/redisenterprise.json

```diff
@@ -0,0 +1,2711 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Name is available"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CheckNameAvailabilityParameters": {
+      "description": "Parameters body to pass for resource name availability check.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type. The only legal value of this property for checking redis enterprise cache name availability is 'Microsoft.Cache/redisenterprise'."
+        }
+      },
+      "required": [
+        "name",
+        "type"
+      ]
+    },
+    "Sku": {
+      "description": "SKU parameters supplied to the create Redis Enterprise cluster operation.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The level of Redis Enterprise cluster to deploy. Possible values: ('GeneralPurpose_G5', 'MemoryOptimized_M20', etc.). For more information on SKUs see the latest pricing documentation. Note that additional SKUs may become supported in the future.",
+          "enum": [
+            "Enterprise_E1",
+            "Enterprise_E5",
+            "Enterprise_E10",
+            "Enterprise_E20",
+            "Enterprise_E50",
+            "Enterprise_E100",
+            "Enterprise_E200",
+            "Enterprise_E400",
+            "EnterpriseFlash_F300",
+            "EnterpriseFlash_F700",
+            "EnterpriseFlash_F1500",
+            "GeneralPurpose_G0",
+            "GeneralPurpose_G1",
+            "GeneralPurpose_G3",
+            "GeneralPurpose_G5",
+            "GeneralPurpose_G10",
+            "GeneralPurpose_G20",
+            "GeneralPurpose_G50",
+            "GeneralPurpose_G100",
+            "GeneralPurpose_G150",
+            "GeneralPurpose_G250",
+            "GeneralPurpose_G350",
+            "GeneralPurpose_G500",
+            "GeneralPurpose_G700",
+            "GeneralPurpose_G1000",
+            "MemoryOptimized_M10",
+            "MemoryOptimized_M20",
+            "MemoryOptimized_M50",
+            "MemoryOptimized_M100",
+            "MemoryOptimized_M150",
+            "MemoryOptimized_M250",
+            "MemoryOptimized_M350",
+            "MemoryOptimized_M500",
+            "MemoryOptimized_M700",
+            "MemoryOptimized_M1000",
+            "MemoryOptimized_M1500",
+            "MemoryOptimized_M2000",
+            "ComputeOptimized_X1",
```

Comment:
>ComputeOptimized_X1 [](http://example.com/codeflow?start=13&length=19)

I thought we're not going to have X1 any more?

Reply:
I see the X1 in our latest checked in code for RP. So included it in here.

Reply:
X1 should be omitted here

Reply:
Removed the X1

Reply:
Thanks for that!